docs/proposals/volume-provisioning.md

```diff
@@ -101,6 +98,12 @@ We propose that:
 
 ### Controller workflow for provisioning volumes
 
+0. Kubernetes administator can configure name of a default StorageClass. This
+   StorageClass instance is then used when user requests a dynamically
+   provisioned volume, but does not specify a StorageClass. In other words,
+   `claim.Spec.Class == ""`
```

Comment:
I talked with our Ops team and they said a key per namespace was not necessary. End user supplied keys was too much.

Instead, 1 key per StorageClass where admins manually create the keys is sufficient.